### DISCUSSION TITLE
Dataset

### DISCUSSION THREAD

**Discussion by AlexBedCarIA** [rramosp/ai4eng.v1#193] [00 - General]
Hola, profe.

Las filas de nuestro dataset son 4446966 y esto nos está dando problemas en colab y en la ejecución de la funciones necesarias para el desarrollo del proyecto, muchas veces se detienen las ejecuciones y se reinicia la conexión con colab o simplemente se tarda muchísimo ejecutando. ¿Qué nos recomienda?

De antemano muchas gracias.

**Comment by rramosp** [comment]
elegid aleatoriamente un subconjunto  y trabajad con ese subconjunto, quizá
1 millón de filas o menos hasta que podáis manejarlo en memoria.

quizá los modelos no den tan bien, pero no importa, lo que importa es el
proceso de experimentación que sigáis

On Mon, Aug 22, 2022 at 8:14 AM AlexBedCarIA ***@***.***>
wrote:

> Hola, profe.
>
> Las filas de nuestro dataset son 4446966 y esto nos está dando problemas
> en colab y en la ejecución de la funciones necesarias para el desarrollo
> del proyecto, muchas veces se detienen las ejecuciones y se reinicia la
> conexión con colab o simplemente se tarda muchísimo ejecutando. ¿Qué nos
> recomienda?
>
> De antemano muchas gracias.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/193>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIL4AT6E7BLHPFJWCMLV2N4LNANCNFSM57HUCGIQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
